### PR TITLE
[FIX] #484 Comment Action for Automated Conference Issues

### PR DESCRIPTION
**Pull request by JohananOppongAmoateng**
Fixed the following:
    The issue isn't actually created by a bot but is created by a user
    The issue could be discovered by using the template
        the title starts with [CONFERENCE] and has the label conference
Potential fix for precommit failure

### CODE REVIEW

File: .github/workflows/conference.yml [hunk 1]
```diff
@@ -53,3 +53,41 @@ jobs:
...
+      - name: Run script
+        env:
+          GITHUB_TOKEN: ${{ env.GITHUB_TOKEN }}
+        run: |
+          python _conferences
+          pre-commit run --all-files --show-diff-on-failure
+
+      - name: Add comment to the issue
+        uses: actions/github-script@v6
+        if: ${{ github.event.issue.user.type == 'User' && startsWith(github.event.issue.title, '[CONFERENCE]') && contains(github.event.issue.labels.*.name, 'conference') }} #adds comment only if the issue is created by a user
```
Comment (kjaymiller) [inline .github/workflows/conference.yml]: I think you can remove the `{{ github.event.issue.user.type == 'User'`

and instead use the "event.action" to be "issue opened" (Not sure what the actual phrase is.

We don't want it to run that las step on edits and closing, when the issue if created.

**Review by kjaymiller — CHANGES_REQUESTED** [review]

File: .github/workflows/conference.yml (hunk 1 above)
Comment (kjaymiller) [reply .github/workflows/conference.yml]: @JohananOppongAmoateng ☝🏾

Comment (JohananOppongAmoateng) [reply .github/workflows/conference.yml]: Noted will work on it

**Comment by JohananOppongAmoateng** [comment]
@kjaymiller please review the changes

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**